app/views/unsubmitted_courses/index.html.haml

```diff
@@ -10,14 +10,10 @@
       .sort-select
         %select.sorts{:rel => "courses"}
           %option{:rel => "asc", :value => "title"}= t("courses.title")
-          %option{:rel => "desc", :value => "revisions"}= t("metrics.revisions")
-          %option{:rel => "desc", :value => "characters"}= t("metrics.word_count")
-          %option{:rel => "desc", :value => "average-words"}= t("metrics.word_count_average")
-          %option{:rel => "desc", :value => "views"}= t("metrics.view")
-          %option{:rel => "desc", :value => "students"}= t("users.editors")
+          %option{:rel => "desc", :value => "instructor"}= t("courses.instructor.one")
+          %option{:rel => "desc", :value => "tags"}= t("metrics.tags")
```

Comment:
Why does tags go before expected students here and vice versa above? Should it be consistent?